src/assets/leaflet/leaflet.css

```diff
@@ -0,0 +1,624 @@
+/* required styles */

```

Comment:
this is already provided in `node_modules/leaflet/dist/leaflet.css`, why is it here too?